sdk/identity/identity/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@azure/identity",
+  "version": "0.1.0",
+  "description": "Provides an implementation of the Azure SDK Client Library for Azure Identity",
```

Comment:
This description definitely implies that `Azure Identity` is a service for which this is the library! Not a blocker for merging this PR, but we should give this some more thought

Reply:
I'll change this to the following and then we can make another pass over how we describe the library before we ship it:

> Provides credential implementations for Azure SDK libraries that can authenticate with Azure Active Directory